src/ng-select/ng-select.types.ts

```diff
@@ -5,7 +5,7 @@ export interface NgOption {
     disabled?: boolean;
     marked?: boolean;
     label?: string;
-    value?: string;
+    value?: string | Object;
```

Comment:
Maybe it make sense to also add resolvedValue so you can precompute pure value if bindValue is used, especially with nested object.